### DISCUSSION TITLE
Analyze Presets for sanger sequencing of single cell TCRalpha and beta chain amplicons

### DISCUSSION THREAD

**Discussion by crystalizee** [milaboratory/mixcr#1424] [Q&A]
Hi,

I did sanger sequencing on multiple 96 wells of single cell TCRalpha and beta chain RT-PCR amplicons, and is trying to use mixcr for the downstream clonotype identifications.

My file names are something like A_01.fastq where the A_01 refers to the well position in 96 well (single cell sorted into A01 well). And this is from A01 to H12.

How can I set the analyze presets so that I can have the clonotypes quantified? I am having trouble making this work because it seems like miXCR is meant for NGS sequencing data. 

Below is the best I've tried up to so far, and it is not giving me the clonotype table I want.

I am attaching some of my own example files for this analysis.

mixcr analyze generic-amplicon \
    --species hs \
    --rna \
    --split-by-sample \
    --floating-left-alignment-boundary \
    --assemble-clonotypes-by VDJRegion \
    --floating-right-alignment-boundary C \
      {{CELL0ROW:a}}_{{CELL0COL:n}}.fastq \
      result

[row_H.zip](https://github.com/milaboratory/mixcr/files/13253875/row_H.zip)

Thank you for the help.

**Comment by mizraelson** [comment]
Hi,
Where does Phred quality come from if its a Sanger sequencing? Usually Sanger output is rather a FASTA file, not FASTQ.
